public/app/core/components/PageNew/Page.tsx

```diff
@@ -84,7 +85,7 @@ export const Page: PageType = ({
           )}
           <div className={styles.pageContent}>
             <CustomScrollbar autoHeightMin={'100%'} scrollTop={scrollTop} scrollRefCallback={scrollRef}>
-              <div className={styles.pageInner}>
+              <div className={cx(styles.pageInner, pageInnerClass)}>
```

Comment:
This and the related change in JasonEditorSettings was fixed in this PR 
https://github.com/grafana/grafana/pull/56524

Reply:
Yeah I merged that PR into mine bc I needed to see my dashboard json